frontend/src/components/chat/markdown-renderer.tsx

```diff
@@ -114,10 +114,6 @@ const CodeBlock = ({ code, language }: CodeBlockProps) => {
   const { theme } = useTheme();
   const [value, setValue] = useState(code);
 
-  useEffect(() => {
-    setValue(code);
-  }, [code]);
```

Comment:
are you sure this is safe to remove? does the code actually update the editor when streamed?